### DISCUSSION TITLE
Items obtained are unusable until Anki is restarted

### DISCUSSION THREAD

**Discussion by Foxy-null** [Unlucky-Life/ankimon#7] [Ideas]

**Comment by Unlucky-Life** [comment]
Right now the item system only works for evolution of pokemon :/. To incorporate all the diffrent functions of the items wil take some time. Stats might be possible to do quickly but the rest might get complicated.

**Comment by Foxy-null** [comment]
I see! Then it's not a problem at the moment. Thank you for letting me know!
(You can close this issue or mark it as "not planned," if you want.)
